src/Treeview.ts

```diff
@@ -300,7 +300,7 @@ export class JsonOutlineProvider implements vscode.TreeDataProvider<string> {
         let element = elementInfo.current.level === 1 ? elementInfo.current : elementInfo.root;
         const keyNode = this.tree && this.tree.getValueAtCharacterIndex(element.key.start, Contains.strict);
         if (keyNode instanceof Json.StringValue) {
-            return keyNode.unquotedValue;
+            return `${keyNode.unquotedValue}${elementInfo.current.level}`;
```

Comment:
Let's use something that can be more strictly parsed, such as "resources[1]".

Also, please update the comment to the function to explain the expected output and usage.

Reply:
Fixed.